iphone/Maps/UI/Editor/MWMEditorViewController.mm

```diff
@@ -965,6 +965,7 @@ - (void)cell:(UITableViewCell *)cell changeSegmented:(YesNoUnknown)changeSegment
           m_mapObject.SetMetadata(feature::Metadata::FMD_DRIVE_THROUGH, "");
           break;
       }
+      break;
```

Comment:
Would it be cleaner to use if instead of the top level switch? It only handles one case.

Reply:
I added as a switch because I expect other metadata will be added there too in the near future:
takeaway=yes/no
delivery=yes/no
toilets=yes/no
changing_table=yes/no